src/otp1/index.ts

```diff
@@ -1,8 +1,9 @@
 import { Router, Request } from 'express'
 import { parseJSON } from 'date-fns'
 import { v4 as uuid } from 'uuid'
+import distance from 'haversine-distance'
```

Comment:
I SDK bruker vi turf for kordinathåndtering. Kan vi bruke @turf/turf-distance for å slippe å ha flere støttebiblioteker som håndterer det samme

```var from = turf.point([-75.343, 39.984]);
var to = turf.point([-75.534, 39.123]);
var options = {units: 'miles'};

var distance = turf.distance(from, to, options);
```
fra https://github.com/Turfjs/turf/tree/master/packages/turf-distance

Reply:
Yes, kan fikse det.

Reply:
Vel, @turf/distance har ikkje TypeScript-støtte (ingen i DefinitelyTyped heller), den drar med seg 4 transitive dependencies, og eg må gjere ein del mapping for å koordinatane på riktig format. Føler det ikkje er verdt det.